### ISSUE TITLE
Profile: edit phone number form does not handle prefilling null area codes

### ISSUE THREAD

**Issue by andaleliz** [department-of-veterans-affairs/va.gov-team#22312]
## What happened?
We have a user on staging (`judy.morrison@id.me`) with a null area code and a country code other than `+1`.  When you attempt to edit the phone number, the null area code isn't handled correctly:
 
![image.png](https://images.zenhubusercontent.com/5d9def68b6de6d00018d50f2/46df0bb4-e43d-41e5-bc77-522072ca8e6b)

## Specs:
- Device: Macbook
- Browser: Chrome
- Test User _(if applicable)_: `judy.morrison@id.me`

## Steps to Reproduce
- URL: Log in as `judy.morrison@id.me` > Select `profile` from the user drop down menu in the main navigation > click the edit button next to the phone number.

- Test User _(if applicable)_: `judy.morrison@id.me`

## Desired behavior
We shouldn't see the word `null` in the form field

## Acceptance Criteria
- [ ] We don't see the word `null` in the form field

## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to Epic** (what body of work is this a part of? possibly `Ongoing Maintenance`)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `triage`, `tools-improvements`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with `Bug`**

**Comment by MarciMcGuireGCIO** [comment]
Zach will work with this and reach out to Erik as needed.

**Comment by andaleliz** [comment]
No, I can't seem to reproduce it. Maybe this was fixed when we made updates to handle international phone numbers. @erikphansen is this issue ringing a bell for you? Any thoughts?

**Comment by erikphansen** [comment]
Okay, so this user no longer has `null` area code so the issue won't come up. And you can't save a user without an area code. But it is conceivable that someone has a phone number on file that has a `null` area code. This _shouldn't_ happen, but it can. Some very small countries do not have area codes. There _shouldn't_ be international area codes saved, but, again, it's not outside the realm of possibility.

But this _is_ still an issue! I was able to confirm it in Cypress by temporarily editing an existing test to modify the `GET user` object to be someone with a `null` area code.

Specifically:
1. I added a `cy.pause();` at line 73 in https://github.com/department-of-veterans-affairs/vets-website/blob/8e9beb66b09cdde8a49a364f90fcc069f2bc50ff/src/applications/personalization/profile/tests/e2e/personal-and-contact-info/modals.cypress.spec.js#L73
1. Then I modified the mock user by changing `areaCode: '503',` to `areaCode: null,` at https://github.com/department-of-veterans-affairs/vets-website/blob/54af4c2bfbf883f37b0b0e808c753f2182dc1501/src/platform/user/profile/vap-svc/util/local-vapsvc.js#L26

Then you can run the test in Cypress and manually enter edit mode:

https://user-images.githubusercontent.com/20728956/138536742-3379ffae-0173-4245-a78d-ae83b390dd64.mp4
